src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketFilePathClient.java

```diff
@@ -42,5 +44,19 @@ public interface BitbucketFilePathClient {
      * @throws ServerErrorException       if the server failed to process the request
      * @throws BitbucketClientException   for all errors not already captured
      */
-    String getFileContent(BitbucketSCMFile scmFile);
+    String getFileContent(BitbucketSCMFile scmFile) throws IOException;
+
+    /**
+     * Retrieve the bytes of a file in a repository. The bytes are encapsulated in an {@link InputStream} object.
+     * @param scmFile the file to retrieve
```

Comment:
newline please.

Also should probably add something about caller being responbile for closing the stream, just to make it clear for future selves.